### PR TITLE
First iteration of filler word removal demo

### PR DESCRIPTION
**Pull request by lazeratops**
This is the first iteration of a filler word removal demo. It has a small frontend and a Python server component. Check out the README for info on how it works and how to run locally. I'll leave some guiding comments below as well.

### CODE REVIEW

**Comment by lazeratops** [comment]
@jessmitch42 Tagging you primarily for your epic front-end-fu!

**Review by jessmitch42 — APPROVED** [review]
Looked at the client-side code. Looks great! Some minor styling suggestions but otherwise no notes. 🚢

File: client/css/header.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+noscript {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  height: 100%;
+  font-family: "Helvetica Neue", Sans-Serif;
+}
```
[the remark this answers is not in the document]
Comment (lazeratops) [reply client/css/header.css:114]: Thank you, that is way better!

**Review by anniebabannie — APPROVED** [review]
Works great!!

File: server/index.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+async def upload_file():
+    """Saves uploaded MP4 file and starts processing.
+    Returns project ID"""
+    files = await request.files
+    file = files["file"]
+    project = Project()
+    file_name = f'{project.id}.mp4'
+    file_path = os.path.join(get_upload_dir_path(), file_name)
+    try:
+        await file.save(file_path)
```
[the remark this answers is not in the document]
Comment (chadbailey59) [reply server/index.py:33]: I think this is totally fine for the sake of a demo; it's much easier to read this way than introducing a lot of async complexity.

**Pull request merged**